src/notebook/widget.ts

```diff
@@ -337,6 +337,25 @@ class NotebookWidget extends Widget {
 }
 
 
+function createElements(tag: string, attr?: any, children?: HTMLElement[] | string): HTMLElement {
```

Comment:
I would put `textContent` as part of `attr` and make children just `HTMLElement[]`, otherwise I think this is a good function to add to `jupyter-js-ui/src/utils`.